units/src/amount/signed.rs

```diff
@@ -96,7 +118,12 @@ impl SignedAmount {
     /// per bitcoin overflows an `i64` type.
     pub const fn from_int_btc_const(whole_bitcoin: i64) -> SignedAmount {
         match whole_bitcoin.checked_mul(100_000_000) {
-            Some(amount) => SignedAmount::from_sat(amount),
+            Some(amount) => {
+                match SignedAmount::from_sat(amount) {
+                    Ok(sa) => sa,
+                    Err(_) => panic!("out of range")
+                }
+            }
```

Comment:
The docs on this function need updating along with this change.

Reply:
good catch, thanks.